app/views/admin/invoices/show.html.erb

```diff
@@ -1,3 +1,7 @@
 <h3> Admin invoice show page </h3>
+<h5><%= @invoice.id %></h5>
+<h5> Status: <%= @invoice.status %></h5>
+<h5> Created on: <%= @invoice.created_at %></h5>
 <h5> Total revenue:</h5>
-<p><%= Invoice.total_revenue(@invoice.id).first.total_revenue %></p>
\ No newline at end of file
+<p><%= Invoice.total_revenue(@invoice.id).first.total_revenue %></p>
```

Comment:
This line could be dried up, there are two different methods named "total_revenue" being called here. I think the invoice model method could be refactored.